pom.xml

```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openmicroscopy</groupId>
+  <artifactId>ome-jai</artifactId>
```

Comment:
Out of curiosity, is there a rationale for going down to `0.x.0` and changing the `artifactId` for this component as opposed to what we have done for other forks like `ome-metakit` where only the `groupId` was modified and the version number chronology was kept?